src/App.tsx

```diff
@@ -1,14 +1,29 @@
 import React, { Component } from "react";
-import { ProofEditor } from "./components/proofEditor/proofEditor";
+import NavBar from "./components/GUI/mainPage/navBar";
+import SiteLayout from "./components/GUI/mainPage/siteLayout";
+import ProofEditor from "./components/GUI/proofEditor/proofEditor";
 
-class App extends Component {
+// tslint:disable-next-line: no-empty-interface
```

Comment:
You don't need empty interfaces; imo `class App extends Component` is clearer than declaring empty interfaces and then disabling the lint rule that forbids that.